### PR TITLE
New Unit Test

### PR DESCRIPTION
**Pull request by C-Frohburg**
- Deleted the redundant unit tests
- Added new unit tests
- Made them easier to run
- Updated readme accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ ADDED VALUE
 
 Unit Tests:
     - In the same directory used to run the initial program run the unit tests with the following command:
-        python -m unittest -v unittests/test_utils.py   
+        python -m unittest
```
Comment (Samuelpf98) [inline README.md:30]: Should we not have commands to run each file of unit tests separately too?

Comment (C-Frohburg) [reply README.md:30]: I can add those commands to the readme

Comment (Samuelpf98) [reply README.md:30]: yeah, just for the sake of completeness

**Review by Lennaayy — APPROVED** [review]
🥇

**Review by Samuelpf98 — APPROVED** [review]
Thanks for updating, looks good now

**Pull request merged**